frontend/src/Map/CrossingMap.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import mapboxstyle from './mapboxstyle';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import ReactMapboxGl, { Layer, Feature } from "react-mapbox-gl";
+
+const Map = ReactMapboxGl({ accessToken: null });
+
+class CrossingMap extends React.Component {
+
+  render () {
+    if (this.props.data.loading) {
+      return (<div>Loading</div>)
+    }
+
+    if (this.props.data.allCrossings == null) {
+      // TODO: add error logging
+      return (<div>Error Loading Crossings</div>);
+    }
+
+    var openCrossings = this.props.data.allCrossings.nodes.filter((crossing) => crossing.statusUpdateByLatestStatusId.statusId === 1);
+    var closedCrossings = this.props.data.allCrossings.nodes.filter((crossing) => crossing.statusUpdateByLatestStatusId.statusId === 2);
+
+    return (
+      <Map
+        style={mapboxstyle}
+        containerStyle={{
+          height: this.props.mapHeight,
+          width: this.props.mapWidth,
+          display: "block"
+        }}
+        center={[ -97.7237, 30.2328 ]}>
```

Comment:
I would recommend to avoid re-creating a ref of the center array at every render, this would trigger unecessary updates because the Map component does an reference check for props `center` and `zoom`. Instead I would suggest to store it in a variable outside of your component render function and use this variable.

Reply:
Thanks for taking a look at this @alex3165 !

I've done a bit more work and now I'm [storing the center in state](https://github.com/cityofaustin/ctxfloods/blob/4d1986014a26b78e235b52be514bfeb8ee352fd8/frontend/src/Map/CrossingMap.js#L69).

I think that matches with your suggestion.